commerce-initializer-util/src/main/java/com/liferay/commerce/initializer/util/CPDefinitionsImporter.java

```diff
@@ -308,19 +307,22 @@ private CPDefinition _importCPDefinition(
 			String imageDependenciesPath, ServiceContext serviceContext)
 		throws Exception {
 
+		Company company = _companyLocalService.getCompany(
+			serviceContext.getCompanyId());
+
+		long companyId = company.getCompanyId();
+		long companyGroupId = company.getGroupId();
```

Comment:
@alecsloan, why declare these as separate variables? Not sure what this gets us

Reply:
No good reason, I think In my head I was thinking "Oh we're using it multiple times, better make a var"